pwnlib/fmtstr.py

```diff
@@ -91,8 +91,14 @@ def send_payload(payload):
 """
 from __future__ import division
 
+import copy
+import functools
 import logging
 import re
+from collections import namedtuple
+from intervaltree import IntervalTree, Interval
+from operator import itemgetter
+from sortedcontainers import SortedList
```

Comment:
Are all those imports needed? It looks like only `SortedList` and `itemgetter` are actually used